eng/pipelines/jobs/pull-request-consistency.yml

```diff
@@ -2,15 +2,9 @@ steps:
   - checkout: self
     submodules: true
 
-  - task: NodeTool@0
-    inputs:
-      versionSpec: "16.x"
-    displayName: Install Node.js
-
-  # purge before install to ensure a clean state between retries
-  - script: node common/scripts/install-run-rush.js install --purge
-    displayName: Install JavaScript Dependencies
-    retryCountOnTaskFailure: 3
+  - template: ../templates/install.yml
+    parameters:
```

Comment:
probably don't need to default to node 16 here, same for the other one

Reply:
Two questions:

1. Are you OK refactoring these to call `templates/install.yml`?  It will make the jobs a little slower since they will (unnecessarily) install .NET, but I thought eliminating the duplicate code was worth it.  We could refactor further if you want to eliminate the .NET install.

2. By removing `Node 16`, you mean just use the template default of `Node 20`?

Reply:
hhm yeah its probably ok to do `1.` if this ever become a problem we can add a flag/refactor later.

and for `2.` yes that what I meant